gamemodes/terrortown/gamemode/client/cl_keys.lua

```diff
@@ -183,11 +183,5 @@ end
 -- @ref https://wiki.facepunch.com/gmod/GM:PlayerButtonUp
 -- @local
 function GM:PlayerButtonUp(ply, btn)
```

Comment:
Why do we need the remaining Hook Stub here?

Reply:
maybe for documentation? I dunno

Reply:
But this is a default hook, we don't need it in our code - https://wiki.facepunch.com/gmod/GM:PlayerButtonUp